cli/internal/terraform/terraform/qemu/modules/instance_group/main.tf

```
@@ -6,10 +6,9 @@ terraform {
     }
   }
 }
-
 resource "libvirt_domain" "instance_group" {
-  name     = "${var.name}-${var.role}-${count.index}"
   count    = var.amount
+  name     = "${var.base_name}-${var.role}-${count.index}"
```

```suggestion
  name     = "${var.base_name}-${var.role}-${local.group_uid}-${count.index}"
```

name should probably also have a uid suffix (similar to GCP and Azure) so different groups use different VM names